support/generate.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -eu
+
```

Comment:
nit: Use a exit function for the cleanup like
```
function cleanup {
  # Clean up temporary files
  rm -rf _obj/ types_gen.go
}
trap cleanup EXIT
```

Reply:
Replacing a single line instruction with a multiple line alternative feels not right. Such clean up functions make sense, if they are more complex. But I don't see the need for this case.

Reply:
I just ended up too often after a failure or a ctrl-c with temporary files on my disk. It is a bit annoying and costs my time. A proper cleanup is good engineering practice.

Reply:
If the intermediate files are not useful for debugging failures (and they don't seem to be) let's add cleanup here like @rockdaboot suggests. The other option would be to add them to `make clean`.

Reply:
Just out of interest, how often are you going to debug the constants so that it is worth it to always keep intermediate files? (My gut feeling says never)